_posts/2016-03-07-shaping-an-open-source-editorial-team.md

```diff
@@ -0,0 +1,74 @@
+---
+
+title: 'Shaping an open source editorial team
+'
+author: Jenn
+layout: blog-post
+comments: false
+
+---
+
+<img src="/blog/images/201603/community.jpg" alt="Community" style="border: 0">
+
+First thing–if you’re reading this and you know an awesome writer friend, share it with them.
+
+Second, if you’re a writer who managed to find their way here, welcome! You’re probably wondering what is an online Hoodie and what the heck does Open Source mean?
```

Comment:
can we make the questions stand out by surrounding them with quotation marks? right now they create a confusing run on sentence.